### PR TITLE
ref(TS) Convert some features to TS

### PR DESCRIPTION
**Pull request by robertpin**

### CODE REVIEW

File: react/features/subtitles/middleware.ts [hunk 1]
```diff
@@ -11,7 +12,7 @@ import {
 import {
     removeTranscriptMessage,
     updateTranscriptMessage
-} from './actions';
+} from './actions.web';
```
Comment (saghul) [inline react/features/subtitles/middleware.ts:15]: If this middleware does not run on mobile, please rename it to middleware.web or this is bound to eventually break.

**Review by saghul — CHANGES_REQUESTED** [review]
1 comment, LGTM otherwise.

File: react/features/subtitles/middleware.ts (hunk 1 above)
Comment (saghul) [reply react/features/subtitles/middleware.ts:15]: It's currently imported on middlewares.any ...

Comment (robertpin) [reply react/features/subtitles/middleware.ts]: Fixed it

**Review by mihhu — APPROVED** [review]

**Pull request merged**